detekt.yml

```diff
@@ -0,0 +1,15 @@
+comments:
+  UndocumentedPublicClass:
+    active: true
+    includes: "**/commonMain/**"
+  UndocumentedPublicProperty:
+    active: true
+    includes: "**/commonMain/**"
+style:
+  UnnecessaryAbstractClass:
+    active: false
+complexity:
+  TooManyFunctions:
+    thresholdInObjects: 21
+    thresholdInClasses: 21
```

Comment:
Not sure if 21 is a reasonable default, any thoughts on this?

Reply:
Detekt already has sensible defaults https://github.com/getsentry/sentry-java/blob/ba90bd9ebf765bac954ce09e164152229da553ef/build.gradle.kts#L102
Any reason why we are being explicit about magic numbers?

Reply:
They probably updated it but the default now includes explicit magic numbers: https://github.com/detekt/detekt/blob/292432870e9382b7b0aad1fb7c17964027d69682/detekt-core/src/main/resources/default-detekt-config.yml#L606

As for TooManyFunctions: the default is 11 so I guess it would be fine to exclude files like SDK entry points?